common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -290,6 +302,9 @@ private void setUpWebView(@NonNull final View view,
         mWebView.getSettings().setUserAgentString(
                 userAgent + AuthenticationConstants.Broker.CLIENT_TLS_NOT_SUPPORTED);
         mWebView.getSettings().setJavaScriptEnabled(true);
+        if (isBrokerRequest && isEstsRequest) {
```

Comment:
What happens once the URL changes within the webview? So we go from eSTS to ADFS for federated auth but the `isEstsRequest` would still say true because it was defined earlier in the flow. This would be a problem

Reply:
Moving the javascript exposure to onPageLoaded()